host/src/containers/GameSessionContainer.tsx

```diff
@@ -89,6 +96,23 @@ const GameSessionContainer = () => {
 
       Promise.all(teamDataRequests)
         .then((responses) => {
+          // if shortAnswer is enabled we need to rebuild the shortAnswerResponses object on refresh
+          if (response.questions[response.currentQuestionIndex].isShortAnswerEnabled === true) {
+            responses.forEach((team) => {
+              team.teamMembers && team.teamMembers.forEach((teamMember) => {
+                teamMember.answers && teamMember.answers.forEach((answer) => {
+                  if (answer.questionId === response.questions[response.currentQuestionIndex].id 
+                    && (response.currentState === GameSessionState.CHOOSE_CORRECT_ANSWER && answer.isChosen) 
+                    || (response.currentState === GameSessionState.CHOOSE_TRICKIEST_ANSWER && answer.isTrickAnswer)
+                  ) {
+                    setShortAnswerResponses((prev) => {
+                      return buildShortAnswerResponses(prev, getQuestionChoices(response.questions, response.currentQuestionIndex), answer, team.name)
```

Comment:
We receive short answer objects on the `host` side via listening to create team answer subscriptions. After running equality checks, we build a short answer responses object and then immediately write that under questions to dynamodb. This keeps the backend in synch with the equality checks we are doing but, because there are no subscriptions to the question, doesn't create any cascading updates across the apps.

Therefore, if our host app crashes, we can read the question object off the backend and get the response object we were building. A more updated version of GameSessionContainer can be found in a later PR, here: https://github.com/rightoneducation/righton-app/pull/850/files#diff-1cd312359a31416a94b68fc30d4b704fed57ea27b70bd4f9ce7a88a186ea2767